tests/helpers.ts

```diff
@@ -20,6 +20,9 @@ import * as nock from 'nock';
 import * as path from 'path';
 
 import * as balenaCLI from '../build/app';
+import { configureBluebird } from '../build/app-common';
+
+configureBluebird();
```

Comment:
`configureBluebird()` must be called before any Bluebird promises are used, including uses by the mocha test framework itself, hence the call as early as possible at module import time.